Mathlib/Tactic/CategoryTheory/BicategoryCoherence.lean

```diff
@@ -295,7 +295,8 @@ syntax (name := whisker_simps) "whisker_simps" : tactic
 elab_rules : tactic
 | `(tactic| whisker_simps) => do
   evalTactic (← `(tactic|
-    simp only [Category.assoc, Bicategory.comp_whiskerLeft, Bicategory.id_whiskerLeft,
+    simp (config := {failIfUnchanged := false}) only [Category.assoc,
```

Comment:
Can you add an explanation of why `failIfUnchanged` is helpful here? Should `whisker_simps` take it's own `config` argument to allow the caller to customize this?

Reply:
It hadn't even occurred to me this counted as a user facing tactic. Mostly it is just used as a component in `coherence`. But sure, I'll give it a `simp` configuration.